test_security/CMakeLists.txt

```diff
@@ -301,29 +289,28 @@ if(BUILD_TESTING)
       "test/test_secure_subscriber.cpp")
 
     set(append_library_dirs "${CMAKE_CURRENT_BINARY_DIR}")
+    set(append_library_dirs "${append_library_dirs}:$ENV{RTI_OPENSSL_LIBS}")
     if(WIN32)
       set(append_library_dirs "${append_library_dirs}/$<CONFIG>")
-    elseif(APPLE)
-      # connext need the openssl libraries to be on the DYLD_LIBRARY_PATH at runtime
-      # as SIP strips these variables on recent MacOS we populate them here
-      # see https://github.com/ros2/ros2/issues/409
-      execute_process(
-        COMMAND "brew" "--prefix" "openssl"
-        RESULT_VARIABLE _retcode
-        OUTPUT_VARIABLE _out_var
-        ERROR_VARIABLE _error_var
-      )
-      if(NOT _retcode EQUAL 0)
-        message(FATAL_ERROR "command 'brew --prefix openssl' failed with error code '${_retcode}' and error message '${_error_var}'")
-      endif()
-      string(STRIP "${_out_var}" _out_var)
-      set(openssl_lib_path "${_out_var}/lib")
-      file(TO_NATIVE_PATH "${openssl_lib_path}" openssl_lib_path)
-      set(append_library_dirs "${append_library_dirs};${openssl_lib_path}")
-    elseif(UNIX)
-      # on bionic, Connext needs RTI's OpenSSL libraries to be on the LD_LIBRARY_PATH
-      # at runtime as the system version of OpenSSL is not supported
-      set(append_library_dirs "${append_library_dirs}:$ENV{RTI_OPENSSL_LIBS}")
+    # TODO(mikaelarguedas) uncomment this once Connext supports OpenSSL 1.1.1
+    # in the meantime use RTI_OPENSSL_LIBS
+    # elseif(APPLE)
+    #   # connext need the openssl libraries to be on the DYLD_LIBRARY_PATH at runtime
+    #   # as SIP strips these variables on recent MacOS we populate them here
+    #   # see https://github.com/ros2/ros2/issues/409
+    #   execute_process(
+    #     COMMAND "brew" "--prefix" "openssl"
+    #     RESULT_VARIABLE _retcode
+    #     OUTPUT_VARIABLE _out_var
+    #     ERROR_VARIABLE _error_var
+    #   )
+    #   if(NOT _retcode EQUAL 0)
+    #     message(FATAL_ERROR "command 'brew --prefix openssl' failed with error code '${_retcode}' and error message '${_error_var}'")
+    #   endif()
+    #   string(STRIP "${_out_var}" _out_var)
+    #   set(openssl_lib_path "${_out_var}/lib")
+    #   file(TO_NATIVE_PATH "${openssl_lib_path}" openssl_lib_path)
+    #   set(append_library_dirs "${append_library_dirs};${openssl_lib_path}")
```

Comment:
Are you keeping the above block commented so that it goes back into use when Connext is updated?

Reply:
https://github.com/ros2/system_tests/pull/409/files/593d7003e929be6a36c8538aaf0eabca13c450a9#diff-6fc729fbfd434d3b129f70533aa0346dR295-R296

yes. Hello it is Friday afternoon.